quickstarts/microsoft.compute/vm-simple-windows/main.bicep

```diff
@@ -100,15 +63,39 @@ param location string = resourceGroup().location
 @description('Name of the virtual machine.')
 param vmName string = 'simple-vm'
 
+@description('Security Type of the Virtual Machine.')
+@allowed([
+  'Standard'
+  'TrustedLaunch'
+])
+param securityType string = 'TrustedLaunch'
+
+@description('Secure Boot setting of the virtual machine. Applicable for securityType TrustedLaunch.')
+param secureBoot bool = true
+
+@description('Custom Attestation Endpoint to attest to. By default, MAA and ASC endpoints are empty and Azure values are populated based on the location of the VM.')
+param maaEndpoint string = ''
```

Comment:
I don't remember off the top of my head, but can you check to see if it is possible to deploy the extension without specifying any endpoint in the request?

Reply:
The deployment succeeds in absence of the entire settings:{} block. However unable to view the attestation status in Portal if deployed without maa endpoint details.

**Deployed**:
![image](https://user-images.githubusercontent.com/18166395/215967437-7f481900-44a3-4000-a148-fbef15055b5a.png)

**Expected**
![image](https://user-images.githubusercontent.com/18166395/215967538-b4ee4001-ce4f-421c-943b-7510043d8cdf.png)

Reply:
OK, in that case can you give the users a set list of endpoints to choose from? If it is freeform, user can enter an incorrect endpoint.

Reply:
@sirfamelin - Added allowed values as discussed, kindly review.